core/utils.js

```diff
@@ -57,13 +57,6 @@ Blockly.utils.removeAttribute = function(element, attributeName) {
 };
 
 /**
- * Cached value for whether 3D is supported
- * @type {boolean}
```

Comment:
Oh I was deleting this because it is also defined in core/blockly.js, but looking around it seems that it is never even used anywhere. Should I remove from both places?